.github/workflows/release.yml

```diff
@@ -49,47 +95,17 @@ jobs:
             }
             core.setFailed(`Draft release not found`)
 
-      - name: Download build artifacts
-        uses: actions/github-script@v6
-        with:
-          script: |
-            let allArtifacts = await github.rest.actions.listWorkflowRunArtifacts({
-               owner: context.repo.owner,
-               repo: context.repo.repo,
-               run_id: context.payload.workflow_run.id,
-            });
-            const matchArtifact = allArtifacts.data.artifacts.filter((artifact) => {
-              return artifact.name.startsWith("policy-server")
-            });
-            for (const artifact of matchArtifact) {
-              let download = await github.rest.actions.downloadArtifact({
-                 owner: context.repo.owner,
-                 repo: context.repo.repo,
-                 artifact_id: artifact.id,
-                 archive_format: 'zip',
-              });
-              let fs = require('fs');
-              file_path = `${process.env.GITHUB_WORKSPACE}/${artifact.name}.zip`;
-              fs.writeFileSync(file_path, Buffer.from(download.data));
-            }
-
-      - name: 'Unzip build artifacts'
-        run: unzip -n "policy-server-*.zip" -d artifacts
-
       - name: Upload release assets
         id: upload_release_assets
         uses: actions/github-script@v6
         with:
           script: |
             let fs = require('fs');
-            // The artifact directory is defined in the previous step. When the
-            // zip files are expanded.
-            let files = fs.readdirSync("artifacts")
-            // Envvar exported in the step para the draft release is found
+            let files = ['policy-server-sbom.spdx', 'policy-server-sbom.spdx.cert', 'policy-server-sbom.spdx.sig']
```

Comment:
I don't think this are the only files, on the release page assets I see there are different ones depending on the arch, plus we have distinct sbom files for the binary and the container image.

I think we should keep them.

I'm suspecting that this informed the previous design; we needed to build the binary out of the Dockerfile, so we could easily gather its SBOMs, per arch.

Then, we built the container image, and gathered its SBOMs, too. Here we were just copying the binary.

Then for the release, we would upload every file in `artifacts/`.


So, 
Is there need for separate SBOMs of the binary (and build process), and the build inside the container (that happens in this PR)?
If we build the bin inside the container stage, that later we discard, do we get the correct SBOMs?

Reply:
Ok, we have to find a way to generate the SBOMs for the binaries. We can't go back to the old build system and keep this change, **unless** we want to end up with something like:
* current `main` branch
* add `Dockerfile.next` which is the `Dockerfile` of this PR and add minimal amount of files needed by it (like the `.cargo/config.toml` - which could also be dropped if we add more `ENV` directives inside of the `Dockerfile.next`)

Let me research a bit how we can approach this problem, without having to go back to the scenario I just illustrated